internal/providers/terraform/google/container_registry.go

```diff
@@ -1,29 +1,22 @@
 package google
 
 import (
+	"github.com/infracost/infracost/internal/resources/google"
 	"github.com/infracost/infracost/internal/schema"
 )
 
-func GetContainerRegistryItem() *schema.RegistryItem {
+func getContainerRegistryItem() *schema.RegistryItem {
 	return &schema.RegistryItem{
 		Name:                "google_container_registry",
 		RFunc:               NewContainerRegistry,
 		ReferenceAttributes: []string{},
 	}
 }
-
 func NewContainerRegistry(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
-	region := d.Get("region").String()
-	components := []*schema.CostComponent{
-		dataStorage(d, u),
-	}
-
-	components = append(components, operations(d, u)...)
-	return &schema.Resource{
-		Name:           d.Address,
-		CostComponents: components,
-		SubResources: []*schema.Resource{
-			networkEgress(region, u, "Network egress", "Data transfer", ContainerRegistryEgress),
-		},
+	r := &google.ContainerRegistry{Address: d.Address, Region: d.Get("region").String(), Location: d.Get("location").String()}
+	if !d.IsEmpty("storage_class") {
```

Comment:
This check isn't needed since it will default to empty anyway, so we can just move this attribute up to be specified on the above line.

Reply:
Right, fixed.